docs/service-mesh.md

```diff
@@ -17,40 +21,44 @@ gcloud container clusters create onlineboutique \
     --machine-type=e2-standard-4 --num-nodes=4
 ```
 
-2. [Install Istio](https://istio.io/latest/docs/setup/getting-started/) on your cluster. 
+1. [Install Istio](https://istio.io/latest/docs/setup/getting-started/) on your
+   cluster.
 
-3. Enable Istio sidecar proxy injection in the `default` Kubernetes namespace. 
+1. Enable Istio sidecar proxy injection in the `default` Kubernetes namespace.
 
-   ```sh
-   kubectl label namespace default istio-injection=enabled
-   ```
+```sh
+kubectl label namespace default istio-injection=enabled
+```
 
-4. Apply all the manifests in the `/release` directory. This includes the Istio and Kubernetes manifests. 
+1. Apply all the manifests in the `/release` directory. This includes the Istio
+   and Kubernetes manifests.
 
-   ```sh
-   kubectl apply -f ./release 
-   ```
+```sh
+kubectl apply -f ./release 
+```
 
-5. Run `kubectl get pods` to see pods are in a healthy and ready state.
+1. Run `kubectl get pods` to see pods are in a healthy and ready state.
 
-6. Find the IP address of your Istio gateway Ingress or Service, and visit the
+1. Find the IP address of your Istio gateway Ingress or Service, and visit the
    application frontend in a web browser.
 
-   ```sh
-   INGRESS_HOST="$(kubectl -n istio-system get service istio-ingressgateway \
-      -o jsonpath='{.status.loadBalancer.ingress[0].ip}')"
-   echo "$INGRESS_HOST"
-   ```
-
-   ```sh
-   curl -v "http://$INGRESS_HOST"
-   ```
-
+```sh
+INGRESS_HOST="$(kubectl -n istio-system get service istio-ingressgateway \
+   -o jsonpath='{.status.loadBalancer.ingress[0].ip}')"
+echo "$INGRESS_HOST"
+```
 
-## Additional service mesh demos using OnlineBoutique 
+```sh
+curl -v "http://$INGRESS_HOST"
+```
 
-- [Canary deployment](https://github.com/GoogleCloudPlatform/istio-samples/tree/master/istio-canary-gke)
-- [Security (mTLS, JWT, Authorization)](https://github.com/GoogleCloudPlatform/istio-samples/tree/master/security-intro)
-- [Cloud Operations (Stackdriver)](https://github.com/GoogleCloudPlatform/istio-samples/tree/master/istio-stackdriver)
-- [Stackdriver metrics (Open source Istio)](https://github.com/GoogleCloudPlatform/istio-samples/tree/master/stackdriver-metrics)
+## Additional service mesh demos using OnlineBoutique
 
+- [Canary
+  deployment](https://github.com/GoogleCloudPlatform/istio-samples/tree/master/istio-canary-gke)
+- [Security (mTLS, JWT,
+  Authorization)](https://github.com/GoogleCloudPlatform/istio-samples/tree/master/security-intro)
+- [Cloud Operations
+  (Stackdriver)](https://github.com/GoogleCloudPlatform/istio-samples/tree/master/istio-stackdriver)
+- [Stackdriver metrics (Open source
+  Istio)](https://github.com/GoogleCloudPlatform/istio-samples/tree/master/stackdriver-metrics)
```

Comment:
If we decide to keep Istio, we should maybe take a look at this:
https://www.aspecto.io/blog/opentelemetry-and-istio-everything-you-need-to-know/

I have never tried myself, but it looks pretty easy to configure.